dev/workflow/rocoto/gefs_tasks.py

```diff
@@ -766,13 +717,8 @@ def arch_tars(self):
         if self.options['do_wave']:
             dep_dict = {'type': 'metatask', 'name': f'{self.run}_wave_post_grid'}
             deps.append(rocoto.add_dependency(dep_dict))
-            dep_dict = {'type': 'metatask', 'name': f'{self.run}_wave_post_pnt'}
+            dep_dict = {'type': 'metatask', 'name': f'{self.run}_wave_stat_pnt'}
```

Comment:
```suggestion
            dep_dict = {'type': 'task', 'name': f'{self.run}_wave_stat_pnt'}
```
`wave_stat_pnt` is not a metatask